bin/steps/python

```diff
@@ -97,6 +97,22 @@ if [ ! "$SKIP_INSTALL" ]; then
   hash -r
 fi
 
+# Heroku uses the get-pip utility maintained by the Python community to vendor Pip.
+# https://github.com/pypa/get-pip
+GETPIP="https://lang-python.s3.amazonaws.com/etc/get-pip.py"
```

Comment:
The SHA should also go in the filename, or you may find there are race conditions when you try to update it.